code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -332,7 +330,7 @@
 			radiation_pulse(location,rad_power)
 
 		var/new_heat_capacity = air.heat_capacity()
-		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY && (air.temperature <= FUSION_MAXIMUM_TEMPERATURE || reaction_energy <= 0))	//If above FUSION_MAXIMUM_TEMPERATURE, will only adjust temperature for exothermic reactions.
```

Comment:
endothermic

Reply:
You're right, I've been thinking of it backwards. The reaction would be endothermic, as in stealing heat from the gas as opposed to the gas as a whole losing heat to the environment and hence being exothermic. Yup. Thanks